x/oracle/keeper/validatorWhiteList_test.go

```diff
@@ -0,0 +1,33 @@
+package keeper_test
+
+import (
+	"log"
+	"testing"
+
+	oracleKeeper "github.com/Sifchain/sifnode/x/oracle/keeper"
+)
+
+func TestKeeper_SetValidatorWhiteList(t *testing.T) {
```

Comment:
This doesn't actually test SetValidatorWhiteList

Reply:
The real test case committed.